dash/include/dash/GlobRef.h

```diff
@@ -130,7 +130,7 @@ class GlobRef
 
   inline bool operator!=(const T & value) const
   {
-    return !(*this != value);
+    return !(*this == value);
```

Comment:
Actually, Clang complained about never-ending recursion in this case. Pretty neat.